### ISSUE TITLE
Not maintained anymore?

### ISSUE THREAD

**Issue by aaronovz1** [baryshev/ect#108]
Is this project still being maintained?

**Comment by HydraOrc** [comment]
No, but it still works perfectly fine if you need something like email templates in your project.
